src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -304,4 +306,24 @@ public void nerfFishing(PlayerFishEvent event) {
       }
     }
   }
+
+  @EventHandler(priority = EventPriority.MONITOR)
+  public void dropItemsOnQuit(PlayerParticipationStopEvent event) {
+    MatchPlayer quitter = event.getPlayer();
+    Player player = Bukkit.getPlayer(quitter.getId());
+
+      for (ItemStack item : quitter.getInventory().getContents()) { //Drop inventory
```

Comment:
No need for the `//` comments in this method, it's self explanatory.

You really only need comments for things that might be harder to understand.

Reply:
Ok! Will remove